tests/OmniSharp.Roslyn.CSharp.Tests/SignatureHelpFacts.cs

```diff
@@ -474,13 +474,40 @@ public Program(Program p, int n)
     {
     }
 }";
-
             var actual = await GetSignatureHelp(source);
             Assert.Equal(3, actual.Signatures.Count());
             Assert.Equal(1, actual.ActiveParameter);
             Assert.Contains("ctor2", actual.Signatures.ElementAt(actual.ActiveSignature).Documentation);
         }
 
+        [Fact]
+        public async Task SignatureHelpForOverloadedMethodsInheritance()
```

Comment:
Can you add a case for methods with the same name that shouldn't be accessible? Eg:
```C#
    class Program
    {
        static void Main(string[] args)
        {
        }

        private void Foo() { }
    }

    class G : Program
    {
        protected void Foo() { }
    }
```